samples/drivers/led_max7219/prj.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_SPI=y
+CONFIG_SPI_1=y
```

Comment:
Please split this into its own boards/nrf52_pca10040.conf as done for 96b_carbon in samples/drivers/led_lpd8806.

Reply:
Fixed